templates/test/running_table.html.ep

```diff
@@ -6,11 +6,11 @@
 	    { targets: 0,
               className: "name" },
             { targets: 3,
+              className: "time",
               "render": function ( data, type, row ) {
-                  console.debug(new Date(data));
-		  if (type === 'display')
-		      return jQuery.timeago(new Date(data + " UTC"));
-		  else
+                    if (type === 'display') {
+		      return jQuery.timeago(new Date(data));
+                  } else
 		      return data;
```

Comment:
I'm completely new to jQuery and almost no experience with JS so bear with me here.
It is ensured that the type will be always 'display' when presenting page? When the data itself are actually returned? The point is if it is always safe to call datetime() on the result of this.